torch/distributed/examples/memory_tracker_example.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import torchvision
+
+from torch.distributed._tools import MemoryTracker
+
+
+def run_one_model(net, input):
+    net.cuda()
+    input = input.cuda()
+
+    # Create the memory Tracker
+    mem_tracker = MemoryTracker()
+    # start_monitor before the training iteration starts
+    mem_tracker.start_monitor(net)
+
+    net.zero_grad(True)
+    loss = net(input)
+    if isinstance(loss, dict):
```

Comment:
nit: I imagine that this `isinstance(loss, dict)` check is determined for the input `resnet34` and tensor (i.e. we know that it is either `True` or `False` always). Is there any reason to include this check in this example?

Reply:
just copy paste from somewhere:), I think it is not harm to add this check, what if loss is empty?